### ISSUE TITLE
A11y - Article listing page - Article card and filters

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1374]
## User story

- Article card
  - Low contrast title
  - Irregular use of list: Card content doesn't fit a list
  - The circle with a letter indicates the type of article and should have a menaningful a11y name (aria-label)

![image](https://github.com/gigascience/gigadb-website/assets/143437854/2e093c2f-7625-42e1-99e3-bfe2f23a1eb0)

- Filters
  - Low contrast: Anything in the filters that has colored text or bg has too low contrast
  - Missing fieldset: Groups of checkboxes should be enclosed in a fieldset

![image](https://github.com/gigascience/gigadb-website/assets/143437854/f10e8b3e-5aa9-4a95-8fbd-8321de7593b4)

https://beta.gigadb.org/search/new?keyword=Genomic&type%5B%5D=dataset&dataset_type%5B%5D=Genomic

## Acceptance criteria

- [x] article title has enough contrast with background
- [x] Card content is not wrapped in a list
- [x] Article type (colored circle with a letter) has meaningful aria-label. NOTE: This is displayed in multiple places across the site so it could be a reusable component
- [ ] ~~Filters text has enough contrast~~. NOTE: Notice that the filter widget has an "old" design, so better wait after newer designs are implemented
- [ ] ~~Enclose checkboxes in fieldset~~. NOTE: Notice that the filter widget has an "old" design, so better wait after newer designs are implemented

**Comment by luistoptal** [comment]
It's hard to see what exactly is the html that render the page without being able to deploy locally as there is similar code among separate files

**Comment by luistoptal** [comment]
Working branch https://github.com/luistoptal/gigadb-website/tree/new-feature/1374/a11y-article-listing-page-article-card-and-filters

**Comment by luistoptal** [comment]
Filters will be handled in #1498
